### PR TITLE
loading node pool

### PR DESCRIPTION
**Pull request by tkallady**

### CODE REVIEW

**Review by DoxasticFox — APPROVED** [review]

File: .terraform/base/gke.tf [hunk 1]
```diff
@@ -81,6 +80,48 @@ resource "google_container_node_pool" "data_nodes" {
...
+    labels = {
+      env = var.project_id
+      type = "data"
+    }
+
+    taint {
+      key = "dataonly"
+      value = "true"
+      effect = "NO_SCHEDULE"
+    }
```
Comment (DoxasticFox) [inline .terraform/base/gke.tf:109]: What does this do?

File: .terraform/base/gke.tf [hunk 2]
```diff
@@ -81,6 +80,48 @@ resource "google_container_node_pool" "data_nodes" {
...
+
+
+
+    # preemptible  = true
+    machine_type = "e2-highcpu-32"
+    disk_size_gb = 20
+    tags         = ["gke-node", "${var.project_id}-gke"]
+    metadata = {
+      disable-legacy-endpoints = "true"
+    }
```
Comment (DoxasticFox) [inline .terraform/base/gke.tf:121]: Why is this needed?

File: .terraform/base/gke.tf (hunk 1 above)
Comment (tkallady) [reply .terraform/base/gke.tf:109]: Prevents pods other than ES being scheduled onto it.

File: .terraform/base/gke.tf (hunk 2 above)
Comment (tkallady) [reply .terraform/base/gke.tf:121]: Not sure, came from some boilerplate
